identities/tests.py

```diff
@@ -47,6 +52,12 @@
         }
     }
 }
+TEST_OPTOUT = {
+    "request_source": "test_source",
+    "requestor_source_id": "1",
+    "address_type": "msisdn",
+    "address": "+27123"
+}
```

Comment:
You definitely want to make this a function (e.g. `mk_optout`) because the way it is now it's a mutable global dictionary that gets messed with by each test and not cleaned up inbetween.